plenum/common/messages/node_messages.py

```diff
@@ -282,10 +281,11 @@ class MessageReq(MessageBase):
     """
     Purpose: ask node for any message
     """
+    allowed_types = {LEDGER_STATUS, CONSISTENCY_PROOF, PREPREPARE,
```

Comment:
Yes, but this is integration tests, and I'm talking about unit tests to support Contracts. Please have a look at the tests at the package I mentioned, it will be very fast to add similar tests there.